assets/js/modules/analytics-4/datastore/settings.js

```diff
@@ -67,6 +68,17 @@ export const INVARIANT_WEBDATASTREAM_ALREADY_EXISTS =
 export const INVARIANT_INVALID_ADS_CONVERSION_ID =
 	'a valid ads adsConversionID is required to submit changes';
 
+export const settings = {
```

Comment:
Let's make this the default rather than a named export for consistency with the others.